src/pages/Mypage/Components/Mainbox/History/History.js

```diff
@@ -0,0 +1,127 @@
+import React, { useCallback, useEffect, useState } from 'react';
+import styled from 'styled-components';
+import { Link } from 'react-router-dom';
+import Item from './Item';
+import { Flex } from '../../../../../styles/Mixin';
+import API from '../../../../../config';
+
+const History = ({ isClicked, setIsClicked }) => {
+  const [HistoryData, setHistoryData] = useState([]);
+
+  const fetchData = useCallback(() => {
+    async function fetchAndSetHistoryData() {
+      const response = await fetch(`${API.History}`, {
+        headers: {
+          Authorization: localStorage.getItem('Authorization'),
+        },
+      });
+      const data = await response.json();
+      setHistoryData(data.result.reservation_list);
+    }
+    fetchAndSetHistoryData();
+  }, [isClicked]);
+
+  useEffect(() => {
+    fetchData();
+  }, [fetchData]);
+
+  return (
+    <HistoryWrap>
+      <HistoryTitle>
+        <img src="/images/mypage/house_icon.png" alt="history" />
+        <h1>내역</h1>
+      </HistoryTitle>
+      <Title>
+        <span>신청내역</span>
+      </Title>
+
+      {HistoryData && HistoryData.length === 0 ? (
+        <HistoryNoneBox>
+          <p>아직 신청한 남의집이 없으시네요!</p>
+          <p>지금 바로 남의집에 놀러가보세요.</p>
+          <Link to="/productlist">
+            <button>남의집 둘러보기</button>
+          </Link>
```

Comment:
```suggestion
          <Link to="/productlist">
          남의집 둘러보기
          </Link>
```

* Link 컴포넌트이지만, a 태그입니다.
* a태그에 중첩해서 button 같은 대화형 컨텐츠를 사용하는건 HTML5의 권장사항이 아닙니다
* 그러면 button 태그에 적용됐던 스타일링이 빠질꺼에요. link 컴포넌트를 스타일링 해주시면 됩니다 :-)
* 자세한 내용은 [여길](https://www.w3docs.com/snippets/css/is-it-possible-to-nest-an-html-button-element-inside-an-a-element-in-html5.html) 참고 하세용

Reply:
넵 수정하였습니다!